From db3a8f47a52bb1ad6232f1536a515a7b1ee5b6f4 Mon Sep 17 00:00:00 2001
From: Alex Leibovich <alexl@marvell.com>
Date: Wed, 29 Dec 2021 10:00:42 +0200
Subject: [PATCH 02/13] thermal: armada: ap806: Thermal values updated

Added correct thermal coefficient for ap806, according to SPEC.

Signed-off-by: Alex Leibovich <alexl@marvell.com>
Change-Id: I4ba96ced66e9cd630758f3d106eaaa7c1b684099
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/68409
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
---
 drivers/thermal/armada_thermal.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/thermal/armada_thermal.c b/drivers/thermal/armada_thermal.c
index 52d63b3997fe..b38fb04c4631 100644
--- a/drivers/thermal/armada_thermal.c
+++ b/drivers/thermal/armada_thermal.c
@@ -629,8 +629,8 @@ static const struct armada_thermal_data armada_ap806_data = {
 	.thresh_shift = 3,
 	.hyst_shift = 19,
 	.hyst_mask = 0x3,
-	.coef_b = -150000LL,
-	.coef_m = 423ULL,
+	.coef_b = -153400LL,
+	.coef_m = 425ULL,
 	.coef_div = 1,
 	.inverted = true,
 	.signed_sample = true,
-- 
2.35.3

